connectivity/drivers/lora/TARGET_STM32WL/STM32WL_LoRaRadio.cpp

```diff
@@ -868,6 +868,10 @@ uint8_t STM32WL_LoRaRadio::get_fsk_bw_reg_val(uint32_t bandwidth)
 {
     uint8_t i;
 
+    if (bandwidth == 0) {
+        return( 0x1F );
```

Comment:
`return 0x1F;` should it be.

Reply:
just this small change and lets start CI

Reply:
Done. 
Does mbed use/have an automatic linter available?

Reply:
we use astyle (you can find details in our workflow guide): https://os.mbed.com/docs/mbed-os/v6.15/contributing/style.html